### PR TITLE
Add missing Tap gateway account entry and 3D Secure setting

### PR DESCRIPTION
**Pull request by edmandiesamonte**

### CODE REVIEW

**Comment by edmandiesamonte** [comment]
please review

File: openapi/components/schemas/GatewayAccountConfig/Tap.yaml [hunk 1]
```diff
@@ -15,3 +15,10 @@ allOf:
             format: password
         required:
           - apiKey
+      settings:
+        type: object
+        description: Tap settings object
+        properties:
+            use3DSecure:
```
Comment (slavcodev) [inline openapi/components/schemas/GatewayAccountConfig/Tap.yaml]:
```suggestion
            enable3DSecure:
```

Just wonder which language is better?

Comment (edmandiesamonte) [reply openapi/components/schemas/GatewayAccountConfig/Tap.yaml]: @slavcodev, in recomm, there's `Enable 3D Secure` option which signifies that we handle the 3Ds process on our end. In Tap's case, they have their own 3D Secure process which can be toggled via just a request flag. I'm actually torn between `use3DSecure` vs `useGateway3DSecure`. The latter says specifically to use the gateway's 3D Secure.

**Comment by edmandiesamonte** [comment]
please review

File: openapi/components/schemas/GatewayAccountConfig/Tap.yaml [hunk 2]
```diff
@@ -15,10 +15,5 @@ allOf:
             format: password
         required:
           - apiKey
-      settings:
```
Comment (vykhrystiuk) [inline openapi/components/schemas/GatewayAccountConfig/Tap.yaml]: Just to clarify. Why has this section been removed in this commit?

Comment (edmandiesamonte) [reply openapi/components/schemas/GatewayAccountConfig/Tap.yaml]: @vykhrystiuk It is no longer needed. Once the MPI was configured, there's already an option in the Advance Settings in recomm to enable 3Ds.

**Pull request merged**